code/game/objects/items/storage/firstaid.dm

```diff
@@ -446,6 +451,18 @@
 	for(var/i in 1 to 7)
 		new /obj/item/reagent_containers/pill/kelotane(src)
 
+/obj/item/storage/pill_bottle/tricordrazine_first_aid
+	name = "bottle of Washes Pain Away pills"
+	desc = "A pill bottle that contains pills that tend all small damages. Its label says: 'Supplement Facts: Tricordrazine 36u. WARN: Do NOT eat more than one pill.'"
+
+/obj/item/storage/pill_bottle/tricordrazine_first_aid/PopulateContents()
+	for(var/i in 1 to 3)
+		new /obj/item/reagent_containers/pill/tricordrazine_first_aid(src)
+
+/obj/item/storage/pill_bottle/tricordrazine_first_aid/medical_doctor/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/reagent_containers/pill/tricordrazine_first_aid(src)
```

Comment:
I know this hard coding is awful, but please understand that this PR's main purpose is not refactoring these codes.